app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  skip_before_filter :require_login
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      auto_login(@user)
+      redirect_to root_path
+      flash[:notice] = 'User was successfully created'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.